.github/workflows/ci.yaml

```diff
@@ -2,6 +2,10 @@ name: Konveyor CI
 
 on: ["push", "pull_request"]
 
+concurrency:
+  group: Konveyor-CI-${{ github.ref }}
+  cancel-in-progress: true
```

Comment:
I'm just curious, this concurency is scoped within the repository that executes the workflow, right? (so it doesn cancel the same workflow executions in different repos)

Reply:
Yes that is my understanding.  Concurrency within this workflow in this repo.